.github/workflows/publish.yml

```
@@ -141,6 +141,7 @@ jobs:
       cache_key: release-${{ needs.prepare.outputs.version }}
       use_cache: false
     secrets: inherit
+    continue-on-error: true
```

actionlint (1.7.7)

when a reusable workflow is called with "uses", "continue-on-error" is not available. only following keys are allowed: "name", "uses", "with", "secrets", "needs", "if", and "permissions" in job "build_windows_x64"

(syntax-check)